.travis.yml

```diff
@@ -202,10 +211,24 @@ install:
         git clone git://github.com/astropy/ci-helpers.git;
         source ci-helpers/travis/setup_conda.sh;
       fi
+    # For APT key updates, see https://ftp-master.debian.org/keys.html
+    - if [ -z $TOXENV ]; then
```

Comment:
this is OK now, but longer term we need to clean up and use more descriptive env variables rather than reuse ones not relevant here.

Reply:
OK, I see the comment above now, so in light of that I go ahead and merge, and leave the possible reshuffle for the future.